elasticdl/python/tests/model_handler_test.py

```diff
@@ -161,6 +173,23 @@ def _get_dataset():
         result = export_model.call(test_data).numpy()
         self.assertEqual(result[0][0], 3.0)
 
+    def test_get_model_with_sparse_to_train(self):
+        model_inst = custom_model_with_sparse_embedding()
+        model_inst = self.model_handler.get_model_to_train(model_inst)
+        self.assertEqual(type(model_inst.layers[1]), Embedding)
+
+    def test_get_model_with_sparse_to_export(self):
+        model_inst = custom_model_with_sparse_embedding()
+        train_model = self.model_handler.get_model_to_train(model_inst)
+        export_model = self.model_handler.get_model_to_export(
+            train_model, dataset=None
+        )
+        test_data = tf.SparseTensor(
+            indices=[[0, 0]], values=[0], dense_shape=(1, 1)
+        )
+        result = export_model.call(test_data).numpy()
+        self.assertEqual(result[0][0], 3.0)
```

Comment:
Please add comments in the test case why the result value is calculated from the parameter value loaded from checkpoint. And the parameter values in the checkpoint can be mentioned.